.ci/ansible/tasks/runners.yml

```diff
@@ -14,6 +14,10 @@
       git_sha: "{{ gitSha | default('Not running on CI')}}"
       repo: "{{repo | default('Not running on CI')}}"
       run_id: "{{runId | default('Not running on CI')}}"
+      division: engineering
+      org: obs
+      team: observability
```

Comment:
Please follow the schema here: https://docs.google.com/document/d/12lYpukxFh7AJVAAR1fHUMxWxab4aZlug5VpiZRDslFg/edit#heading=h.iuj1t86zt3oh